astropy/wcs/tests/helper.py

```diff
@@ -14,7 +14,7 @@ def __init__(
         cdelt=[1, 1],
         pc={"PC1_1": 1, "PC2_2": 1},
     ):
-        """set essential parameters of the model (coord transformations)"""
+        """set essential parameters of the model (coord transformations)."""
```

Comment:
I think a colon is more appropriate here. This is a string comment and ending it with a dot is not correct, IMO.

Reply:
```suggestion
        # set essential parameters of the model (coord transformations):
```

Reply:
✅. I turned the docstring into a comment.